ui/src/views/ZaaktypeEditView.tsx

```diff
@@ -1,177 +1,180 @@
-          horizontal: 'right',
-        },
-      });
-    } else {
-      navigateQuit();
-    }
+    setLoading(false);
   };
 
   return (
-    <Stack direction={'row'} flexWrap={'wrap'} width={'100%'} spacing={spacings.xlarge} useFlexGap>
-      <Stack
-        direction={'row'}
-        justifyContent={'space-between'}
-        alignItems={'center'}
-        flexWrap={'wrap'}
-        width={'100%'}
-        spacing={spacings.large}
-        useFlexGap
-      >
-        <Typography variant="h1">
-          {loadingState ? <Skeleton variant="text" width={200} /> : value?.zaaktype?.omschrijving}
-        </Typography>
-        <Stack direction={'row'} spacing={spacings.small}>
-          <Button variant="outlined" onClick={handleQuit}>
-            Annuleren
-          </Button>
-          <Button variant="contained" onClick={handleSave} disableElevation>
-            Opslaan
-          </Button>
-        </Stack>
-        <Stack spacing={spacings.small} width="100%">
-          <Typography variant="h2">
-            {loadingState ? (
-              <Skeleton variant="text" width={320} />
-            ) : (
-              'Koppel informatieobjecttypen'
-            )}
-          </Typography>
-          <Typography variant="body2">
-            {loadingState ? (
-              <Skeleton variant="text" width={340} />
-            ) : (
-              'Hier kunt u de gekoppelde informatieobjecttypen wijzigen.'
-            )}
-          </Typography>
-          <BulkEditor
-            apiRef={apiRef}
-            loading={loadingState}
-            zaaktype={value?.zaaktype!}
-            informatieobjecttypen={value?.informatieobjecttypen!}
-          />
-        </Stack>
-      </Stack>
-    </Stack>
+    <List
+      errors={errors}
+      fields={[
+        'omschrijving',
+        {
+          editable: true,
+          name: 'richting',
+          type: 'string',
+          options: [
+            { label: 'Inkomend', value: 'inkomend' },
+            { label: 'Uitgaand', value: 'uitgaand' },
+            { label: 'Intern', value: 'intern' },
+          ],
+        },
+        {
+          editable: true,
+          name: 'statustype',
+          type: 'string',
+          options:
+            zaaktype?.statustypen.map((s) => ({ label: s.omschrijving, value: s.url })) || [],
+        },
+        { editable: true, name: 'volgnummer', type: 'number' },
+        'beginGeldigheid',
+        'eindeGeldigheid',
+      ]}
+      formProps={{
+        labelSubmit: 'Opslaan',
+        onSubmit: handleSave,
+      }}
+      selectable={true}
+      selected={selectedRows as AttributeData[]}
+      labelSelect="(de)selecteer rij"
+      labelSelectAll="(de)selecteer {countPage} rijen"
+      title={<>Bewerken{loading ? <Outline.ArrowPathIcon spin={true} /> : null}</>}
+      loading={loading}
+      objectList={(objectList || []) as AttributeData[]}
+      onSelectionChange={(s) => setSelectedRows(s as ZaakTypeEditViewRowT[])}
+      sort={true}
```

Comment:
Maybe there is a way to sort so that the selected IOT come first? Also due to the colours, they are a bit difficult to see:
![Screenshot from 2024-04-09 16-54-46](https://github.com/maykinmedia/open-zaaktypebeheer/assets/19154114/4caed157-da33-4465-b554-2b256c63e3f4)